go/pkg/bertyprotocol/store_metadata_index.go

```diff
@@ -167,7 +167,7 @@ func (m *metadataStoreIndex) handleGroupAddDeviceSecret(event proto.Message) err
 	}
 
 	if m.ownMemberDevice.device.Equals(senderPK) {
-		m.sentSecrets[string(e.DestMemberPK)] = struct{}{}
+		m.sentSecrets[string(e.DevicePK)+string(e.DestMemberPK)] = struct{}{}
```

Comment:
Why are you making this change?
This map only concerns our current device and is only modified after checking that the device key of the sender matches.